.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -365,3 +369,24 @@ stages:
         - template: ./templates/.builder-release-template.yaml
           parameters:
             artifactName: 1804-fips-gen2-gpu-containerd 
+  - stage: build_vhd_1804_arm64_gen2_containerd
+    dependsOn: []
+    condition: eq('${{ parameters.build1804arm64gen2containerd }}', true)
+    jobs:
+    - job: build
+      timeoutInMinutes: 180
+      steps:
+        - bash: |
+            echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
+            echo '##vso[task.setvariable variable=OS_SKU]Ubuntu'
+            echo '##vso[task.setvariable variable=OS_VERSION]18.04'
+            echo '##vso[task.setvariable variable=HYPERV_GENERATION]V2'
+            echo '##vso[task.setvariable variable=AZURE_VM_SIZE]Standard_D8darm_V3'
```

Comment:
Does using `Standard_D8darm_V3` need any additional testing/changes elsewhere? Asking since other builds use `Standard_DS2_v`2 and I wasn’t able to see `Standard_D8darm_v3` vm size used in Agentbaker.

Reply:
no, the packer pipeline will use the vm size that we ask it to use and it creates a VM and do the installations. For now the ARM SKU is only visible on the subscriptions (1 MSFT subscription and 1 AME subscription) that the VHD build pipeline is using.